go-selinux/selinux_linux_test.go

```diff
@@ -25,7 +25,8 @@ func TestSetFileLabel(t *testing.T) {
 	out.Close()
 	defer os.Remove(tmp)
 
-	if err := SetFileLabel(tmp, con); err != nil {
+	err = SetFileLabel(tmp, con)
+	if err != nil {
```

Comment:
I don't understand the reason for this change.

Reply:
not a huge one; my editor's linter warns about variables that are shadowed; in this case an `err` already existed. I could just remove the `:`, but thought it's more clear to break it in two lines if we're reusing the variable.

Reply:
So, in this case the linter warning should be ignored. Yes, we have a variable named `err` already, and yes, here we create another `err` which shadows the original one. No, there is no issue here.

Or maybe just remove the `:`.